setup.cfg

```diff
@@ -51,7 +51,7 @@ install_requires =
     eralchemy==1.2.*
     hieroglyph>=2.1.0
     setuptools>=50
-    sphinx>=4.4,!=7.2.*
```

Comment:
Might as well switch to `sphinx>=7` at this stage, I'm not sure whether this (and other changes) is back compat as far as Sphinx 4.